tests/test_logical_types.py

```diff
@@ -137,6 +137,39 @@ def test_not_null_date():
     assert (data2['date'] == datetime.date(2017, 1, 1))
 
 
+schema_datetime = {
+    "fields": [
+        {
+            "name": "timestamp-millis",
+            "type": {'type': 'long', 'logicalType': 'timestamp-millis'}
+        },
+        {
+            "name": "timestamp-micros",
+            "type": {'type': 'long', 'logicalType': 'timestamp-micros'}
+        }
+    ],
+    "namespace": "namespace",
+    "name": "name",
+    "type": "record"
+}
+
+
+# particularly critical on Windows
+# see https://github.com/fastavro/fastavro/issues/389
+def test_ancient_datetime():
+    data1 = {
+        'timestamp-millis': datetime.datetime(1960, 1, 1),
+        'timestamp-micros': datetime.datetime(1960, 1, 1)
+    }
+    binary = serialize(schema_datetime, data1)
+    data2 = deserialize(schema_datetime, binary)
+
+    assert data1['timestamp-millis'] == data2['timestamp-millis'].replace(
+        tzinfo=None)
```

Comment:
What does the `replace(tzinfo=None)` do?

Reply:
It turns a tz-aware datetime into a naive one, by simply remove the timezone. Quick example:

```
UTC = pytz.UTC
d_utc = datetime.datetime(2001, 1, 1, 0, 0, 0, 0, UTC)
d_naive = datetime.datetime(2001, 1, 1, 0, 0, 0, 0)

assert d_utc != d_naive
assert d_utc.replace(tzinfo=None) == d_naive
```

In this test, the data2 datetimes have the fastavro UTC tzinfo, so removing it allows for a simple comparison with data1.